traits/etsconfig/tests/test_etsconfig.py

```diff
@@ -8,7 +8,10 @@
 import sys
 import tempfile
 import time
-import unittest
+if sys.version_info[:2] == (2, 6):
+    import unittest2 as unittest
+else:
+    import unittest
```

Comment:
The library code in `traits.etsconfig` can't depend on anything else in `traits`, but the tests don't matter.